static/src/javascripts/projects/common/modules/ui/autoupdate.js

```diff
@@ -0,0 +1,242 @@
+// @flow
+
+import bean from 'bean';
+import bonzo from 'bonzo';
+import qwery from 'qwery';
+
+import $ from 'lib/$';
+import fastdom from 'lib/fastdom-promise';
+import fetchJSON from 'lib/fetch-json';
+import detect from 'lib/detect';
+import mediator from 'lib/mediator';
+import twitter from 'common/modules/article/twitter';
+import { Sticky } from 'common/modules/ui/sticky';
+import { scrollToElement } from 'lib/scroller';
+import RelativeDates from 'common/modules/ui/relativedates';
+import NotificationCounter from 'common/modules/ui/notification-counter';
+import { checkElemsForVideos } from 'common/modules/atoms/youtube';
+
+type autoUpdateOptions = {
+    toastOffsetTop: number,
+    minUpdateDelay: number,
+    maxUpdateDelay: number,
+    backoffMultiplier: number,
+};
+
+const autoUpdate = (opts?: autoUpdateOptions) => {
+    const options = Object.assign(
+        {
+            toastOffsetTop: 12,
+            minUpdateDelay:
+                (detect.isBreakpoint({ min: 'desktop' }) ? 10 : 30) * 1000,
+            maxUpdateDelay: 20 * 60 * 1000, // 20 mins
+            backoffMultiplier: 0.75, // increase or decrease the back off rate by modifying this
+        },
+        opts
+    );
+
+    // Cache selectors
+    const $liveblogBody = $('.js-liveblog-body');
+    const $toastButton = $('.toast__button');
+    const $toastText = $('.toast__text', $toastButton);
+    const toastContainer = qwery('.toast__container')[0];
```

Comment:
Why is this using `qwery` when all the other selectors use `$`. Can we drop `qwery` from this module?